config/initializers/wisper_event_subscriptions.rb

```diff
@@ -1,8 +1,5 @@
 # Initializer to register event listeners via the Wisper (https://github.com/krisleech/wisper) gem
 
-#Monitor when school has been regenerated so we can show school target progress report
-Wisper.subscribe(Targets::ContentGenerationListener.new, :ContentBatch)
-
```

Comment:
need to do this as part of content batch job when all school jobs have finished

school_target = school.most_recent_target
school_target.update(report_last_generated: Time.zone.now) if school_target.present?